src/Sylius/Bundle/ApiBundle/Resources/config/serialization/PaymentRequest.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<serializer xmlns="http://symfony.com/schema/dic/serializer-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd"
+>
+    <class name="Sylius\Component\Payment\Model\PaymentRequest">
+        <attribute name="id" ignore="true" />
+
```

Comment:
```suggestion
```

Reply:
It seems to be needed, otherwise we get:
```
request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Invalid identifier value or configuration." at ReadListener.php line 89 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Invalid identifier value or configuration. at /Users/gsadee/Sites/Sylius/Sylius/vendor/api-platform/core/src/Symfony/EventListener/ReadListener.php:89)\n[previous exception] [object] (ApiPlatform\\Metadata\\Exception\\InvalidIdentifierException(code: 0): Parameter \"id\" not found, check the identifiers configuration. at /Users/gsadee/Sites/Sylius/Sylius/vendor/api-platform/core/src/State/UriVariablesResolverTrait.php:41)"} []
```